students/JonathanMauk/lesson06/mailroom_test.py

```diff
@@ -0,0 +1,49 @@
+import os
+import unittest
+import mailroom04 as mailroom
+
+
+class Mailroom04Test(unittest.TestCase):
+
+    # Tests that donor_db looks like it should.
+    def test_donor_db(self):
+        donor_db = {'John Smith': [18774.48, 8264.47, 7558.71], 'Jane Doe': [281918.99, 8242.13],
+                        'Alan Smithee': [181.97, 955.16], 'Tom D.A. Harry': [67.1, 500.98], 'Joe Shmoe': [200.01]}
+        self.assertEqual(mailroom.donor_db, donor_db)
+
```

Comment:
This test isn't really necessary. I'd recommend defining `donor_db` as a dict literal in your mailroom code rather than zipping together the lists to form the dict so there isn't any room for error